http-server/src/test/resources/clientcert-java/clientcert.sh

```diff
@@ -26,3 +26,23 @@ openssl pkcs12 -name client -inkey client.key -in client.crt -export -passout pa
 # create client truststore
 keytool -import -noprompt -alias ca -file ca.crt -storetype pkcs12 -storepass airlift -keystore client.truststore
 
+
+#Creating the replacement keystore files for both the server and the client
```

Comment:
nit : Let's be more descriptive - ` A kesytore for testing the KeyStoreScanner's ability to transparently update the ssl config without needing a restart`